### PR TITLE
Possibly improved answer to Exploring More ls Arguments #477

### PR DESCRIPTION
**Pull request by jarekbryk**
After writing this up I may be slightly leaning towards @rgaiacs suggestion to remove the `ls -lh` from the lesson entirely. Is this really what very beginner users need to be aware of?

### CODE REVIEW

File: _episodes/02-filedir.md [hunk 1]
```diff
@@ -833,10 +833,48 @@ and we will see it in many other tools as we go on.
...
 > > ## Solution
-> > The `-l` arguments makes `ls` use a **l**ong listing format, showing not only
-> > the file/directory names but also additional information such as the file size
-> > and the time of its last modification. The `-h` argument makes the file size
-> > "**h**uman readable", i.e. display something like `5.3K` instead of `5369`.
+> > The `-l` argument makes `ls` use a **l**ong listing format, showing not only
+> > the file or directory names but also additional properties of each file or directory. 
+> > Consider this example output of `ls -l` in the `shell-novice` folder:
+
+`-rwxr-xr--   1 nelly  staff   690  5 Jan 16:02 README.md`
```
Comment (rgaiacs) [inline _episodes/02-filedir.md]: You can't break the quote block here. Use

~~~
> > ~~~
> > -rwxr-xr--   1 nelly  staff   690  5 Jan 16:02 README.md
> > drwxr-xr-x   6 nelly  staff   204  5 Jan 16:02 assets
> > drwxr-xr-x  13 nelly  staff   442  5 Jan 16:02 bin
> > ~~~
~~~

File: _episodes/02-filedir.md [hunk 2]
```diff
@@ -833,10 +833,48 @@ and we will see it in many other tools as we go on.
 > nevertheless.
 >
 > > ## Solution
-> > The `-l` arguments makes `ls` use a **l**ong listing format, showing not only
-> > the file/directory names but also additional information such as the file size
-> > and the time of its last modification. The `-h` argument makes the file size
-> > "**h**uman readable", i.e. display something like `5.3K` instead of `5369`.
+> > The `-l` argument makes `ls` use a **l**ong listing format, showing not only
+> > the file or directory names but also additional properties of each file or directory.
```
Comment (rgaiacs) [inline _episodes/02-filedir.md]: Why use "properties" instead of "information"?

File: _episodes/02-filedir.md [hunk 3]
```diff
@@ -833,10 +833,48 @@ and we will see it in many other tools as we go on.
...
+> > Consider this example output of `ls -l` in the `shell-novice` folder:
+
+`-rwxr-xr--   1 nelly  staff   690  5 Jan 16:02 README.md`
+`drwxr-xr-x   6 nelly  staff   204  5 Jan 16:02 assets`
+`drwxr-xr-x  13 nelly  staff   442  5 Jan 16:02 bin`
+
+> > The fields displayed for each file are, from the left: file mode, number of links, owner name, 
+> > group name, number of bytes in the file, abbreviated month, day-of-month, hour 
+> > and minute of the last modification of the file and the file/folder name.
+
```
Comment (rgaiacs) [inline _episodes/02-filedir.md]: Don't break the quote block.

File: _episodes/02-filedir.md [hunk 4]
```diff
@@ -833,10 +833,48 @@ and we will see it in many other tools as we go on.
...
+> > 1. 	The **file mode** field is made up of 10 characters. The the first character indicates
```
Comment (rgaiacs) [inline _episodes/02-filedir.md]: Use space instead of tab. ;-)

**Comment by jarekbryk** [comment]
Thanks for the comments - will fix this, but I am not sure how (this is the first time I get a comment on a PR) - do I change the things you suggest and then submit another PR?

File: _episodes/02-filedir.md (hunk 2 above)
Comment (gdevenyi) [inline _episodes/02-filedir.md]: Agree, while "properties" is more technically correct, information is more understanding

Comment (jarekbryk) [inline _episodes/02-filedir.md]: "Properties" is used in line 831, just above, but I agree with @gdevenyi

**Comment by shwina** [comment]
While I think this is a nice explanation for `ls -lh` (and I am glad that you got to work on your first PR with feedback, @jarekbryk!), I'm feeling very much like this is not required for the lesson. I've done the lesson several times without having learners stress out about what the output of `-lh` really is. @rgaiacs @gdevenyi ?

**Comment by gdevenyi** [comment]
Hrm, I think this amount of detail may be overkill for the purposes of this lesson. While I love the detail, many of these points are things that most learners won't (possibly ever) need to know.

I think this material is covered in the shell-extras under permissions and such.

Thanks for your contribution!

**Pull request closed without merging**